app/services/GetMemberStatesService.scala

```diff
@@ -33,12 +34,16 @@ class GetMemberStatesService @Inject()(connector: GetMemberStatesConnector)
   def getMemberStatesSelectItems()(implicit hc: HeaderCarrier): Future[Seq[SelectItem]] = {
     connector.getMemberStates().map {
       case Left(_) => throw MemberStatesException("No member states retrieved")
-      case Right(value) => value.map { memberState =>
-        SelectItem(
-          value = Some(memberState.countryCode),
-          text = s"${memberState.country} (${memberState.countryCode})"
-        )
-      }
+      case Right(value) =>
+        value
+          .filterNot(_.countryCode == GB_PREFIX)
+          .filterNot(_.countryCode == NI_PREFIX)
```

Comment:
I think doing filterNot twice would have some (although negligible) performance degradation. Could this be combined into a single `filterNot`?

Reply:
Addressed on https://github.com/hmrc/emcs-tfe-create-movement-frontend/pull/298/commits/c35448b32cea9d38790d2eece2072a4a8eb204c3